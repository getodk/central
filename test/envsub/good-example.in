# Alone:
${SIMPLE}

# end of line
Simple value: ${SIMPLE}

# Missing curlies:
Should not be substituted: $SIMPLE

# Start of line:
${SIMPLE} and then more content.

# This is not a sub variable:
$simple

# Two values on the same line:
first: ${SUBVAL_1}, second: ${SUBVAL_2}!

# Two values on the same line, both repeated:
first: ${SUBVAL_1}, ${SUBVAL_1}, ${SUBVAL_1}, second: ${SUBVAL_2} ${SUBVAL_2}!
