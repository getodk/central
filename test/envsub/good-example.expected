# Alone:
sv_simple

# end of line
Simple value: sv_simple

# Missing curlies:
Should not be substituted: $SIMPLE

# Start of line:
sv_simple and then more content.

# This is not a sub variable:
$simple

# Two values on the same line:
first: sub_val_one, second: sub_val_two!

# Two values on the same line, both repeated:
first: sub_val_one, sub_val_one, sub_val_one, second: sub_val_two sub_val_two!
